src/main/java/com/openshift/internal/restclient/model/template/Template.java

```diff
@@ -56,4 +56,14 @@ public Template(ModelNode node, IClient client, Map<String, String []> propertyK
 		return resources;
 	}
 
+	@Override
+	public void updateParameterValues(Collection<IParameter> parameters) {
+		Map<String, IParameter> actuals = getParameters();
+		for (IParameter param : parameters) {
+			if(actuals.containsKey(param.getName())) {
```

Comment:
@adietish add a boolean  to determine if updated?  We can. I'm not opposed.  What's the value?